doc/tutorials/namespaces.md

```diff
@@ -99,74 +100,104 @@ Having namespaces enables both admins and users to set specific parts of the app
 
 ## How it Works on the Command Line (kdb) ##
 
-Let's say your app requires the following configuration data:
+We will provide an example of how you can configure [elektrified](../help/elektra-glossary.md) applications.
 
-- **/sw/org/myapp/policy** - a security policy to be applied
-- **/sw/org/myapp/default_dir** - a place where the application stores its data per default
+Our exemplary application will be the key database access tool `kdb` as this should already be installed on your system.
 
-We now want to enter this configuration by using the **kdb** tool.
+`kdb` can be configured by the following configuration data:
 
-The security policy will most probably be set by your system administrator.
-So she enters
+- **/sw/elektra/kdb/#X/PROFILE/verbose** - sets the verbosity of kdb
+- **/sw/elektra/kdb/#X/PROFILE/quiet** - if kdb should suppress non-error messages
 
+**X** and **PROFILE** are placeholders for the _major version number_ and the _profile_ to which this configuration applies. If we want to set configuration for the default profile we can set PROFILE to %. The name of the key follows the convention described [here](../help/elektra-key-names.md).
+
+Say we want to set `kdb` to be more verbose in the current directory by default. In this case we have to set _verbose_ to 1 in your dir namespace.
 ```sh
-sudo kdb set "system/sw/org/myapp/policy" "super-high-secure"
-#> Create a new key system/sw/org/myapp/policy with string super-high-secure
+kdb set dir/sw/elektra/kdb/#0/%/verbose 1
```

Comment:
Was explained in the section before. I have added some more explanation, nevertheless.